elastic/datadog_checks/elastic/elastic.py

```diff
@@ -40,6 +41,10 @@ def __init__(self, name, init_config, instances):
         # Host status needs to persist across all checks
         self.cluster_status = {}
 
+        if self.instance.get('auth_type') == 'aws' and self.instance.get('url'):
+            self.HTTP_CONFIG_REMAPPER = self.HTTP_CONFIG_REMAPPER.copy()
+            self.HTTP_CONFIG_REMAPPER['aws_host'] = urlparse(self.instance['url']).hostname
```

Comment:
1. Are we sure this `aws_host` always match the url hostname? Seems this doesn't give user any choice implemented this way.

2. And it seems a bit magic to assign `aws_host` automatically. 

If we really do so, a) `aws_host` being derived from `url` should be documented on the config yaml, and b) `HTTP_CONFIG_REMAPPER` could be moved to outside the class to avoid reassigning it (creates confusion).

Reply:
1. It appears so, based on https://github.com/DataDog/integrations-core/pull/5249#discussion_r359740768
2. I'll change to account for user choice
3. We do https://github.com/DataDog/integrations-core/blob/e599a6b33ad676aa2a35884a45c1438e9b3bf09d/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L152

Reply:
Cool.

About `HTTP_CONFIG_REMAPPER`, could be moved to outside the class to avoid reassigning it (creates confusion).